### PR TITLE
Add new banner, content update

### PR DESCRIPTION
**Pull request by zprood**
## Done

[Add new banner, content update]

### CODE REVIEW

**Comment by anthonydillon** [comment]
screenshot_20180907-085936__01

**Comment by zprood** [comment]
@anthonydillon re-write the two sections, highlight the image.

File: templates/about/index.html [hunk 1]
```diff
@@ -8,13 +8,35 @@
...
 
 <div class="p-strip is-bordered">
   <div class="row">
-    <div class="col-8">
+    <div class="col-7">
       <h1>关于Ubuntu</h1>
       <p>ubuntu |oǒ'boǒntoō|</p>
-      <p>Ubuntu 是一个古老的非洲语单词，意思是“Humanity to Others”（以人道善待他人），同时还有着“群在故我在”的意味。Ubuntu 操作系统将 Ubuntu 精神带到了计算机世界。 Ubuntu 也是Canonical公司的产品和所有的品牌商标，而科能（上海）软件科技有限公司则是Canonical公司在中国注册的子公司，公司总部位于英国伦敦。</p>
+      <p>Ubuntu 是一个古老的非洲语单词，意思是“Humanity to Others”（以人道善待他人），同时还有着“群在故我在”的意味。Ubuntu 操作系统将 Ubuntu 精神带到了计算机世界。 </p>
+      <blockquote class=""><p>Ubuntu 也是<a href="https://wwww.canonical.com" class="p-link--external">Canonical</a>公司的产品和所有的品牌商标，而科能（上海）软件科技有限公司则是Canonical公司在中国注册的子公司，公司总部位于英国伦敦。
```
Comment (anthonydillon) [inline templates/about/index.html]: Missing classes: https://docs.vanillaframework.io/en/patterns/pull-quote

File: templates/cloud/index.html [hunk 2]
```diff
@@ -32,12 +32,19 @@
...
       <p class="p-heading--four">70％的公有云工作负载和54％的OpenStack云<a href="#fn1" id="r1">*</a></p>
       <p><a href="https://www.ubuntu.com/download/cloud" class="p-button--positive">安装OpenStack</a></p>
-      <p>如您已明确企业的需求，可直接<a href="https://pages.ubuntu.com/contact-us-in-chinese.html" class="p-link--external">联系我们</a>获得技术支持以及认证服务。</p>
+      <br>
+      <p>如您已明确企业的需求，可直接<a href="https://pages.ubuntu.com/contact-us-in-chinese.html" class="p-link--external" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact Us', 'eventLabel' : 'cn.ubuntu.com - cloud', 'eventValue' : undefined });">联系我们</a>获得技术支持以及认证服务。</p>     
     </div>
+
+    <div class="col-5">
+      <img src="https://assets.ubuntu.com/v1/17a7067f-451bib.jpg">
+      <p>打破私有云经济学的迷思，Canonical的托管私有云优于公有云。<a href="https://pages.ubuntu.com/Multi-cloudEbook_chinese.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'ebook', 'eventLabel' : 'cloud-ebook', 'eventValue' : undefined });">查看分析机构451 Research的研究报告。</a></p>
```
Comment (anthonydillon) [inline templates/cloud/index.html]: External link missing the external class

File: templates/cloud/index.html [hunk 3]
```diff
@@ -505,35 +513,39 @@ <h2>MAAS：裸机即服务</h2>
...
       </ul>
-      <p><a href="https://www.ubuntu.com/server/maas">进一步了解MAAS&nbsp;›</a></p>
-    </div>
-    <div class="col-2 p-hero__item">
-      <img class="p-hero__image u-image-position--bottom" src="https://assets.ubuntu.com/v1/8ca2eab9-hero-ui-image.png" alt="MAAS UI web page showing node listing" style="max-height:100%;">
-    </div>
-  </div>
+
+      <ul class="p-list">
+      <li><img src="https://assets.ubuntu.com/v1/8ca2eab9-hero-ui-image.png" alt="MAAS UI web page showing node listing"></li>
```
Comment (anthonydillon) [inline templates/cloud/index.html]: The link should have a `p-list__item`

File: templates/cloud/index.html [hunk 4]
```diff
@@ -505,35 +513,39 @@ <h2>MAAS：裸机即服务</h2>
...
+      </ul>
+      <p><a href="https://www.ubuntu.com/server/maas">进一步了解MAAS&nbsp;›</a></p>
```
Comment (anthonydillon) [inline templates/cloud/index.html]: This is an external link and therefore should have the external class an not the `&nbsp;>`

File: templates/cloud/index.html [hunk 5]
```diff
@@ -505,35 +513,39 @@ <h2>MAAS：裸机即服务</h2>
...
-    </div>
-  </div>
+
+      <ul class="p-list">
+      <li><img src="https://assets.ubuntu.com/v1/8ca2eab9-hero-ui-image.png" alt="MAAS UI web page showing node listing"></li>
+      </ul>
+      <p><a href="https://www.ubuntu.com/server/maas">进一步了解MAAS&nbsp;›</a></p>    
+   
+               </div>
+          </div>
```
Comment (anthonydillon) [inline templates/cloud/index.html]: Missing a closing div here

File: templates/cloud/index.html [hunk 6]
```diff
@@ -505,35 +513,39 @@ <h2>MAAS：裸机即服务</h2>
...
+        <li class="p-list__item is-ticked">信息丰富的监控功能</li>        
       </ul>
-      <p><a class="p-link--external" href="https://landscape.canonical.com/">进一步了解Landscape</a></p>
-    </div>
-    <div class="col-2 p-hero__item">
-      <img class="p-hero__image u-image-position--bottom" src="https://assets.ubuntu.com/v1/80429220-landing.png" alt="Landscape landing page screenshow" style="max-height:100%;">
-    </div>
-  </div>
+      <ul class="p-list">
+        <li><img src="https://assets.ubuntu.com/v1/80429220-landing.png" alt="Landscape landing page screenshow"></li>
```
Comment (anthonydillon) [inline templates/cloud/index.html]: Missing `p-list__item`

File: templates/cloud/index.html [hunk 7]
```diff
@@ -505,35 +513,39 @@ <h2>MAAS：裸机即服务</h2>
...
+      </ul>
+      <p><a href="https://www.ubuntu.com/server/maas">进一步了解MAAS&nbsp;›</a></p>
```
Comment (anthonydillon) [inline templates/cloud/index.html]: Replace the > with an external link class please

File: templates/cloud/index.html [hunk 8]
```diff
@@ -505,35 +513,39 @@ <h2>MAAS：裸机即服务</h2>
...
-      <img class="p-hero__image u-image-position--bottom" src="https://assets.ubuntu.com/v1/80429220-landing.png" alt="Landscape landing page screenshow" style="max-height:100%;">
-    </div>
-  </div>
+      <ul class="p-list">
+        <li><img src="https://assets.ubuntu.com/v1/80429220-landing.png" alt="Landscape landing page screenshow"></li>
+      </ul>
+      <p><a href="https://www.ubuntu.com/server/maas">进一步了解MAAS&nbsp;›</a></p>    
+   
+               </div>
+          </div>
```
Comment (anthonydillon) [inline templates/cloud/index.html]: Missing closing div here

File: templates/contact/index.html [hunk 9]
```diff
@@ -40,11 +40,11 @@ <h1>联系Ubuntu</h1>
       <h2>商业合作咨询</h2>
       
       <ul class="p-list--divided">
-  <li class="p-list__item">物联网及设备 <a class="u-float--right" href="https://pages.ubuntu.com/contact-us-in-chinese.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact Us', 'eventLabel' : 'cn.ubuntu.com - contact-iot', 'eventValue' : undefined });">我要咨询</a></li>
+  <li class="p-list__item">物联网及设备 <a class="u-float--right" href="https://pages.ubuntu.com/contact-us-in-chinese.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact Us', 'eventLabel' : 'cn-contact-iot', 'eventValue' : undefined });">我要咨询</a></li>
```
Comment (anthonydillon) [inline templates/contact/index.html]: External link

File: templates/contact/index.html [hunk 10]
```diff
@@ -40,11 +40,11 @@ <h1>联系Ubuntu</h1>
...
 
 
 
-  <li class="p-list__item">云及服务器 <a class="u-float--right" href="https://pages.ubuntu.com/contact-us-in-chinese.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact Us', 'eventLabel' : 'cn.ubuntu.com - contact-cloud', 'eventValue' : undefined });">我要咨询</a></li> 
+  <li class="p-list__item">云及服务器 <a class="u-float--right" href="https://pages.ubuntu.com/contact-us-in-chinese.html" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Contact Us', 'eventLabel' : 'cn-contact-cloud', 'eventValue' : undefined });">我要咨询</a></li>
```
Comment (anthonydillon) [inline templates/contact/index.html]: Add external link class please

File: templates/desktop/index.html [hunk 11]
```diff
@@ -332,14 +332,14 @@ <h4>作为个人电脑</h4>
       <h4>Snapcraft</h4>
       <p>snap是全新的Linux应用格式包，拥有更安全、稳定、易维护更新等特点。snap由snapcraft打包而来，现在就开始制作自己的snap吧！</p>
       <p>
-        <a href="http://snapcraft.io">开始制作自己的snap</a>
+        <a href="http://snapcraft.io" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'snapcraft', 'eventLabel' : 'snapcraft-cnsite', 'eventValue' : undefined });">开始制作自己的snap</a>
```
Comment (anthonydillon) [inline templates/desktop/index.html]: This is an external link

File: templates/desktop/index.html [hunk 12]
```diff
@@ -332,14 +332,14 @@ <h4>作为个人电脑</h4>
...
       </p>
     </div>
     <div class="col-4 p-divider__block">
       <h4>Snapcraft论坛</h4>
       <p>在打包snap的过程中，如你有遇到相关技术问题，都可以前往snapcraft论坛进行搜索和咨询。欢迎您的到来，让我们做得更好。</p>
       <p>
-        <a href="http://forum.snapcraft.io">立即前往</a>
+        <a href="http://forum.snapcraft.io" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'snapcraft', 'eventLabel' : 'snapcraft-cnsite', 'eventValue' : undefined });">立即前往</a>
```
Comment (anthonydillon) [inline templates/desktop/index.html]: Please update the event category and add the external class

File: templates/index.html [hunk 13]
```diff
@@ -117,7 +73,7 @@ <h4 class="p-heading-icon__title p-muted-heading">研讨会</h4>
         </div>
       </div>
       <h3 class="p-heading--four">
-        <a class="p-link--external" href="https://www.brighttalk.com/webcast/6793/311297">如何在Orange Pi上构建snap商店与Ubuntu Core技术介绍</a>
+        <a class="p-link--external" href="https://pages.ubuntu.com/Cloud_K8s_WBN_CN_landing-page.html"  onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'webinar', 'eventLabel' : 'k8s-webinar', 'eventValue' : undefined });">K8s：Canonical Kubernetes网络研讨会（录制视频）</a>
```
Comment (anthonydillon) [inline templates/index.html]: External link

**Comment by anasereijo** [comment]
- our new default h1 font is `thin` now - can this be updated across the website?

### Homepage takeover:
can we please replace the openstack logo with [this pictogram](https://assets.ubuntu.com/v1/a7916513-picto-openstack.svg)? please make sure there is some white space between the copy and the illustration too.

### About page:
- the illustration should be "full height" (maybe set as a background image?) so that the dots will look like the are coming from the edges of that section.

### Cloud index:
- can change the takeover background to look like the english version? you can check the html and css [here](https://github.com/canonical-websites/www.ubuntu.com/pull/3307)

### Desktop index:
please delete the last (blank) section - see image below:
ubuntu

### Internet of Things index:
- we updated the screenshot on ubuntu.com. Please use [this image](https://assets.ubuntu.com/v1/30a4ee8e-branded+store+snaps.jpg)

File: templates/cloud/index.html (hunk 2 above)
Comment (zprood) [reply templates/cloud/index.html]: fixed all the missing external classes, will push soon.

**Comment by zprood** [comment]
@anasereijo Hi,

Homepage takeover: _Done_
About page: _Done_
Cloud index:  Can we use the current one ? it's more commercial for OpenStack ebook. 
Desktop index: _Done_
Internet of Things index: _Done_

All are set in my local environment, there're some problems that I can't push new changes, will find why it happens.

**Pull request closed without merging**